app/controllers/concerns/hydra/collections_controller_behavior.rb

```diff
@@ -142,6 +142,7 @@ def collection_params
 
     # Queries Solr for members of the collection.
     # Populates @response and @member_docs similar to Blacklight Catalog#index populating @response and @documents
+    # TODO: this should be moved into Hydra::Collections::SearchBehaviors
```

Comment:
I agree with that too. What I meant was that it seems like this logic should be handled by the SearchBuilder, such as `add_collection_members_to_query` or something. But, no, it wouldn't be called directly by the controller.